pages/give-devices.js

```diff
@@ -2,28 +2,21 @@ import Head from 'next/head';
 import React from 'react';
 import { useRouter } from 'next/router';
 
+import CountryData from '../data/country-data.json';
 import Footer from '../components/Footer';
 import InputFormControl from '../components/form/InputFormControl';
 import Navbar from '../components/Navbar';
 import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
 // Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
-const COUNTRIES = [
-  { value: 'US', label: 'United States' },
-  { value: 'CA', label: 'Canada' },
-  { value: 'MX', label: 'Mexico' },
-  { value: 'DE', label: 'Germany' },
-  { value: 'LV', label: 'Latvia' },
-  { value: 'IN', label: 'India' },
-  { value: 'BD', label: 'Bangladesh' },
-  { value: 'PH', label: 'Philippines' },
-];
+
+CountryData.map((value) => ({ label: value }));
```

Comment:
This line is no longer required.

```suggestion
```

This is because the `map` method only returns a new array, it doesn't affect the original array, so here we're creating a new array but it doesn't get stored anywhere. You found an alternative way of doing this that didn't require the map in the end.

[Array.map](https://www.w3schools.com/jsref/jsref_map.asp)

Reply:
Oh, yes I will remove this.